pkg/apis/pipeline/v1alpha1/image_resource.go

```diff
@@ -19,6 +19,8 @@ package v1alpha1
 import (
 	"fmt"
 	"strings"
+
+	corev1 "k8s.io/api/core/v1"
 )
 
 // NewImageResource creates a new ImageResource from a PipelineResource.
```

Comment:
might be an opportunity to remove the Image resource till its properly implemented

Reply:
I had a private chat with @bobcatfish about this. I am rephrasing @bobcatfish point here. There is value is keeping image resource because tasks can use its attributes. For eg: in templating params.